### PR TITLE
fixed margin issue in reading 

### PR DESCRIPTION
**Pull request by ttimbers**
and added foreword-text.Rmd to `pdf/_bookdown.yml` (this should have been done in #412).

### CODE REVIEW

**Comment by ttimbers** [comment]
Closes #385

File: reading.Rmd [hunk 1]
```diff
@@ -1066,6 +1082,10 @@ population_nodes <- html_nodes(page, selectors)
 head(population_nodes)
 ```
 
+```{r echo = FALSE}
+print_html_nodes(head(population_nodes))
```
Comment (trevorcampbell) [inline reading.Rmd:1086]: This did not work in the HTML version -- attached is a screenshot of what this output

scren
s

Comment (trevorcampbell) [reply reading.Rmd:1086]: checking PDF now...

Comment (trevorcampbell) [reply reading.Rmd:1086]: Seems to work fine in PDF. Not sure why there's a difference!

File: reading.Rmd [hunk 2]
```diff
@@ -6,6 +6,22 @@ library(magrittr)
 library(knitr)
 knitr::opts_chunk$set(message = FALSE,
                       fig.align = "center")
+
+print_html_nodes <- function(html_nodes_object) {
+  if(!is_latex_output()) {
+    html_nodes
```
Comment (ttimbers) [inline reading.Rmd:12]:
```suggestion
    html_nodes_object
```

Comment (ttimbers) [reply reading.Rmd:12]: This should fix the error you spotted.

**Comment by trevorcampbell** [comment]
looks good to me now!

**Pull request merged**